cmd/tsbs_generate_queries/databases/redistimeseries/common.go

```diff
@@ -0,0 +1,45 @@
+package redistimeseries
+
+import (
+	"time"
+
+	"github.com/timescale/tsbs/cmd/tsbs_generate_queries/uses/devops"
+	"github.com/timescale/tsbs/cmd/tsbs_generate_queries/utils"
+	"github.com/timescale/tsbs/query"
+)
+
+
+// BaseGenerator contains settings specific for Cassandra database.
```

Comment:
this files contains some references to Cassandra.

Reply:
solved on the new commit